consensus/polybft/polybft_test.go

```diff
@@ -54,7 +54,7 @@ func TestPolybft_VerifyHeader(t *testing.T) {
 		Number:    0,
 		ExtraData: append(make([]byte, signer.IstanbulExtraVanity), genesisExtra.MarshalRLPTo(nil)...),
 	}
-	genesisHeader.ComputeHash()
+	_ = genesisHeader.ComputeHash()
```

Comment:
?

Reply:
I just wanted to stress out that I intentionally don't use return value from `ComputeHash` function. I thought it is a good coding practice in Go?
However if it bothers you I can revert it back, don't mind it either way. :slightly_smiling_face: